main.nf

```diff
@@ -127,23 +138,23 @@ Channel
     .fromList(params.samples_to_process)
     .map { sm ->
         def rg_arg = ""
-        return tuple(sm.path, sm.orig_id, sm.sm_id, rg_arg, null, null, sm.sample_type, sm.read_length)
+        return tuple(sm.bam, indexFile(sm.bam), sm.orig_id, sm.sm_id, rg_arg, null, null, sm.sample_type, sm.read_length)
```

Comment:
**question (non-blocking)**: Out of curiosity, did including the index as an input improve performance?

Reply:
`mosdepth` errors if no index is given